docs/reference/commandline/images.md

```diff
@@ -343,3 +344,34 @@ b6fa739cedf5        committ                   latest
 746b819f315e        postgres                  latest
 {% endraw %}
 ```
+
+### Customize sort order
+
+To customize list orders, you can use `--sort-by` option, which takes the format
+of `field[:order]`. `field` is one or more of: "repository",
+"tag", "digest", "createdat", "id" and "size". `order` should be either `asc`(ascendant)
+or `desc`(descendant).
+The meaning of each `field` is described in the [`Formatting`](#formatting) section.
+
+If you specify an invalid `field` or `order` string, a hint will be printed, such as the following:
+
+```bash
+$ docker images --sort-by invalid
+can't sort by "invalid", it must be in [repository,tag,digest,createdat,id,size]
```

Comment:
Ooops, I forgot to update this. I've updated error message in codes but forget to update related docs. 

Updating now...